### DISCUSSION TITLE
Will the v4 eink screen work?

### DISCUSSION THREAD

**Discussion by Nexsq** [jayofelony/pwnagotchi#125] [Q&A] [answered]
I'm curious if the waveshare eink v4 display will work without problems with the latest version (v2.8.9)

**Comment by jayofelony** [comment, marked as answer]
Yes it works

**Reply by Nexsq** [reply]
Do you also know if i can add the battery indicator plugin for the pisugar v2 on the same version somehow? I know it works with alluminium sugar

**Reply by Nexsq** [reply]
Also, thanks for help

**Reply by jayofelony** [reply]
> Do you maybe also know if i can add the battery indicator plugin for the pisugar v2 on the same version (v2.8.9) somehow? I know it works on alluminium sugar

No idea
